client/blocks/login/error-notice.jsx

```diff
@@ -61,6 +61,11 @@ class ErrorNotice extends Component {
 			return null;
 		}
 
+		// The existing_wpcom_user error is handled inline in LoginForm
```

Comment:
Agreed, the comment could be more helpful - I've updated it to clarify how the error is being handled.